server/modules/listeners/listeners.module.ts

```diff
@@ -290,40 +290,8 @@ export class ListenersModule {
 			notifications.notifyUserInThisInstance(userId, 'callabandoned', { queuename, queuedcallafterabandon });
 		});
 
-		service.onEvent('notify.desktop', (uid, notification): void => {
-			notifications.notifyUserInThisInstance(uid, 'notification', notification);
-		});
-
```

Comment:
Is there a reason for removing these listeners?

Reply:
It was overwritten by mistake. I have fixed the issue.